tests/src/python/test_qgsappstartup.py

```diff
@@ -110,21 +110,21 @@ def testOptionsPath(self):
         if sys.platform[:3] == 'dar':  # Mac
             subdir = 'qgis.org'
         ini = os.path.join(subdir, 'QGIS2.ini')
-        for p in ['test_opts', 'test opts', u'test_optsé€']:
+        for p in ['test_opts', 'test opts', 'test_optsé€']:
```

Comment:
I don't think that this kind of things will works with Python2.
In Python 3.3, the `u` prefix has been reintroduced. :-)
http://pythonhosted.org/six/#six.u